instrumentation/log4j/log4j-context-data/log4j-context-data-2.17/javaagent/build.gradle.kts

```diff
@@ -28,7 +28,7 @@ testing {
     val testDisableThreadLocals by registering(JvmTestSuite::class) {
       sources {
         groovy {
-          setSrcDirs(listOf("src/test/groovy"))
+          setSrcDirs(listOf("src/test/groovy")).exclude("AutoLog4jBaggageTest.groovy")
```

Comment:
Instead of doing this (and instead of adding a new `testAddBaggage` task), you could create a new test suite with a new source folder (e.g. `src/testBaggage/groovy`) that gets completely separate configuration & classpath from the other suites.

Reply:
ah yeah that seems easier to maintain - made the change

I could do the same for the other build.kts but i'm finding it slow to make gradle changes in intellij with all the re-syncing

Reply:
Yeah, this project is super heavy on IntelliJ...

It'd be great if you could do the same in the other modules too -- this one was a must have, since we already had suites, and it requires hacks to make it work.

Reply:
i'll flake on the gradle changes and look for some other issues to help with 🙂